sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveTableValuedFunctions.scala

```diff
@@ -105,7 +105,7 @@ object ResolveTableValuedFunctions extends Rule[LogicalPlan] {
 
   override def apply(plan: LogicalPlan): LogicalPlan = plan resolveOperators {
     case u: UnresolvedTableValuedFunction if u.functionArgs.forall(_.resolved) =>
-      builtinFunctions.get(u.functionName) match {
+      builtinFunctions.get(u.functionName.toLowerCase) match {
```

Comment:
@hvanhovell Just did a search. Do we still need to check the case sensitivity setting of the analyzer in this case?

Based on [the Hive doc](https://cwiki.apache.org/confluence/display/Hive/LanguageManual+UDF), 
> All Hive keywords are case-insensitive, including the names of Hive operators and functions.
 
Our regular function names are always case insensitive. See the [code](https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala#L74)